src/longopt/grdcut_inc.h

```diff
@@ -25,29 +25,27 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 		  short_directives,    long_directives,
 		  short_modifiers,     long_modifiers,
 		  transproc_mask */
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "",
-		  GMT_TP_STANDARD },
-	{ 0, 'F', "",
-	          "",                  "",
+	{ 0, 'D', "dryrun",
 	          "",                  "",
+	          "t",                 "trailer",
 		  GMT_TP_STANDARD },
-	{ 0, 'G', "",
-	          "",                  "",
+	{ 0, 'E', "extract",
```

Comment:
I think `rowslice` & `colslice` is better here.

Reply:
Well, **-E** can be **--extract** and then the two directives can be columns or row, e.g.

```
--extract=column:54:30W
--extract=row:30N
```

No? We are not replacing -E with two new long-options.